javascript/javascript2/week3/fastest.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="fastest.css"></head>
+<body>
+    <div id="two-Btn">
+    <input type="text" id="numberAsTimes"><br><br>
```

Comment:
How about `id="duration"`?  :) For type, you could use `number` so there'd be no need for `isNaN` in the .js file